src/MicrosoftTeams.ts

```diff
@@ -875,6 +881,12 @@ namespace microsoftTeams
         channelId?: string;
 
         /**
+         * The developer-defined unique id for the entity this content points to.
+         * If none was provided, this value is generated from the contentUrl.
+         */
+        entityId: string;
```

Comment:
We'll always provide it. It may just be a hash of the content url though at this point, but it'll always be given.